streams/src/main/java/org/apache/kafka/streams/processor/internals/SinkNode.java

```diff
@@ -44,6 +44,14 @@
         this.partitioner = partitioner;
     }
 
+    public Serializer getKeySer() {
```

Comment:
nit: `getKeySerde()` -- similar below

Reply:
Since only Serializer is returned, I didn't include 'de' in the method name.

Reply:
I see. Than I would prefer `getKeySerializer()` (it's a nit, but I like to avoid abbreviations)